components/results_table/wikis/starcraft2/results_table_custom.lua

```diff
@@ -0,0 +1,280 @@
+
+	return mw.html.create()
+		:node(header)
+		:node(row)
+end
+
+function CustomResultsTable._allKillMatch(args)
+	local teamName = args.team or TBD
+
+	local match = DivTable.create():setStriped(true)
+		:addClass('general-collapsible collapsed inherit-bg')
+		:css('width', '100%')
+		:css('margin', 0)
+		:css('padding', 0)
+		:row(
+			DivTable.HeaderRow()
+				:addClass('inherit-bg')
+				:cell(mw.html.create('div'):css('width', '35%'):wikitext('vs [[' .. teamName .. ']]'))
+				:cell(mw.html.create('div'):css('width', '30%'):wikitext(''))
+				:cell(mw.html.create('div'):css('width', '35%'):node(GeneralCollapsible.DefaultToggle()))
+		)
+
+	local mapIndex = 1
+	local validRound = true
+	while validRound do
+		validRound = CustomResultsTable._allKillMapRow(args, 'm' .. mapIndex, match)
+		mapIndex = mapIndex + 1
+	end
+
+	CustomResultsTable._allKillMapRow(args, 'ace', match)
+	CustomResultsTable._allKillMapRow(args, 'ace1', match)
+	CustomResultsTable._allKillMapRow(args, 'ace2', match)
+	CustomResultsTable._allKillMapRow(args, 'ace3', match)
+
+	return match:create()
+end
+
+function CustomResultsTable._storeAllKill(args)
+	local numberOfAllKills = tonumber(Variables.varDefault('allKills')) or 0
+	if not Namespace.isMain() or numberOfAllKills == 0 then
+		return
+	end
+
+	mw.ext.LiquipediaDB.lpdb_datapoint('allKill_' .. mw.title.getCurrentTitle().text, {
+		type = 'allkills',
+		information = numberOfAllKills,
+	})
+end
+
+function CustomResultsTable._allKillMapRow(args, prefix, match)
+	if not (args[prefix .. 'p1'] or args[prefix .. 'p2'] or args[prefix .. 'win'] or args[prefix .. 'walkover'])then
+		return
+	end
+
+	local opponentLeft = mw.html.create('div'):css('text-align', 'right')
+	local mapCell = mw.html.create('div'):css('text-align', 'center')
+	local opponentRight = mw.html.create('div'):css('text-align', 'left')
+
+	if tonumber(args[prefix .. 'win']) == 1 or tonumber(args[prefix .. 'walkover']) == 1 then
+		opponentLeft:addClass('bg-win'):css('font-weight', 'bold')
+	elseif tonumber(args[prefix .. 'win']) == 2 or tonumber(args[prefix .. 'walkover']) == 2 then
+		opponentRight:addClass('bg-win'):css('font-weight', 'bold')
+	elseif args[prefix .. 'win'] == 'skip' then
+		opponentLeft:css('text-decoration', 'line-through')
+		mapCell:css('text-decoration', 'line-through')
+		opponentRight:css('text-decoration', 'line-through')
+	end
+
+	local map
+	if args[prefix .. 'walkover'] then
+		map = 'Walkover'
+	else
+		map = args[prefix .. 'map']
+		map = map and ('[[' .. map .. ']]') or 'TBA'
+	end
+
+	match:row(
+		DivTable.Row():addClass('should-collapse inherit-bg')
+			:cell(opponentLeft:node(CustomResultsTable._opponentDispaly(args, prefix, 1)):css('width', '33%'):done())
+			:cell(mapCell:wikitext(map):css('width', '34%'):done())
+			:cell(opponentRight:node(CustomResultsTable._opponentDispaly(args, prefix, 2)):css('width', '33%'):done())
+	)
+
+	return true
+end
+
+function CustomResultsTable._opponentDispaly(args, prefix, side)
```

Comment:
Function name type

Reply:
don't get this comment

Reply:
Dispaly should be Display

Reply:
haha didn't see that even when re reading it several times